src/mk.sh

```diff
@@ -489,6 +503,26 @@ function kernel_deploy
   fi
 }
 
+# This function retrieves and prints information related to the building
+# process of the kernel.
+function build_info()
+{
+  if [[ -f ".config" ]]; then
+    local kernel_name=$(get_kernel_release "$flag")
+    local kernel_version=$(get_kernel_version "$flag")
+    local compiled_modules=$(grep -c "=m" .config)
+
+    say "Kernel source information"
+    echo -e "\tName: $kernel_name"
+    echo -e "\tVersion: $kernel_version"
+    echo -e "\tTotal modules to be compiled: $compiled_modules"
+
+  else
+    complain ".config file was not found. You have to be inside a kernel"\
+    "repository with a .config file to run this command"
+  fi
```

Comment:
We don't need to complain about anything if the user does not have a local `.config` file; we just don't want to try the grep command.